ncm-modprobe/src/main/pan/components/modprobe/schema.pan

```diff
@@ -11,7 +11,8 @@ type module_type = {
     "alias"     ? string # alias for the loadable module"
     "options"   ? string # options for the loadable module"
     "install"   ? string # command to run when loading module"
-    "remove"    ? string # command to run when removing module "
+    "remove"    ? string # command to run when removing module"
+    "blacklist" ? string # ignored, only the name of the module is used"
```

Comment:
Should this not be a boolean instead? (Not sure what setting it to false should do, since setting it to false would be the equivalent of setting it to null?)